v2/car.go

```diff
@@ -0,0 +1,46 @@
+package car
+
+const prefixBytesSize = 16
```

Comment:
why do we have these constant lower case `prefixBytesSize` that are separate from the exported variables?


I think i'd weakly encourage exported, pre-calculated consts so that there's no danger of them being changed